### ISSUE TITLE
Narrowing failures of 4.1.1 duplicate id's

### ISSUE THREAD

**Issue by DavidMacDonald** [w3c/wcag21#15]
I think there is a fair amount of agreement that many identical id circumstances don't affect accessibility. For WCAG next do we want to limit that failure to circumstances where the id is referenced by something that shows up in the Accessible API, such as aria references, and HTML labels etc.

**Comment by awkawk** [comment]
David, this sounds like a Silver issue since we can't change that WCAG 2.0 says "and any IDs are unique". 

Would you agree?  If so, we should close this with the "defer to Silver" label.
